pages/tidal tools/sourcecode.md

```diff
@@ -1,120 +1,107 @@
 
-## Perform the analysis
+### Standard mode
 
-When analyzing a source code (or multiple), you have two options.
-
-1. Perform the source code analysis and upload the result immediately to your workspace. This is Tidal Tools' default behaviour. To do so, all you need to do is run the following command.
+1. Perform the source code analysis and upload the result immediately to your workspace:
 
    ```bash
    cd /path/to/source-code
    tidal analyze code --app-id [app_id_for_your_application]
    ```
 
-   Alternatively, you can pass multiple locations or even wildcard for analysis. You can even specify individual files.
+   Alternatively, you can pass multiple locations or even wildcard paths, or filenames for analysis. This will result in all the provided and matching file paths to be considered source code for the one application you are analyzing.
 
    ```bash
    tidal analyze code [/path/to/source-code-A] [/path/to/source-code-B] --app-id [app_id_for_your_application]
    ```
 
-2. Perform the source code analysis and upload the results **at a later time**. The following section will explain how to run database analysis in **offline mode**
-
-### Running offline
+### Offline mode
 
-There are circumstances in which you need to perform a source code analysis on an environment without or with restricted internet access. In such case, you can perform the analysis, capture its results and at a later stage upload those results to your Tidal workspace.
+If the machine running code analysis can not access the internet; you can perform the analysis and store the results locally and upload those results to your workspace from a different machine. This process will involve two machines, one that has internet access and one that has no internet access, and a copy of the application's source code.
 
-These are the steps you need to follow in order to bypass internet access limitations:
+1. On a machine with internet access, [install, and configure Tidal Tools](#what-you-need-to-get-started).
 
-1. On a Machine with internet access, you need to install, and configure Tidal Tools.
-
-2. Package Tidal Tools and its required docker container images into a tar file. This will allow you to move the archive file into your restricted environment. To do so, run the following command.
+2. Backup Tidal Tools required dependencies into a backup file.
 
    `tidal backup`
 
-   Once it has finished, you will find (in your current location) a tar file called `tidal-snapshot_DATE.tar`. This is the file you need to transport into your internet restricted environment.
+   When finished in the current working directory there will be a file called `tidal-snapshot_DATE.tar`. Move this file, in any reasonable way you can, to the machine with the code on it to be analyzed
 
-3. On the machine that has no internet access, you will now restore Tidal Tools using the following command.
+3. On the machine that has no internet access, you will now restore the backup:
```

Comment:
Aside: backup is a confusing term for users. "I don't want to backup anything. Why can't I just save and load?"

Reply:
yea good call, i noticed that when writing it too. current action verbs are from the viewpoint of the software vs. the user.